pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -822,6 +822,28 @@ FROM parse_test ORDER BY id ASC
 0101000000000000000000F87F000000000000F87F  0101000020E6100000000000000000F87F000000000000F87F  0101000020E6100000000000000000F87F000000000000F87F  0020000001000010E67FF80000000000007FF8000000000000
 NULL                                        NULL                                                NULL                                                NULL
 
+subtest ST_PointFromGeoHash
+query T
+SELECT ST_AsText(ST_PointFromGeoHash('s000000000000000'))
+----
+POINT (0 0)
+
+query T
+SELECT ST_AsText(ST_PointFromGeoHash('kkqnpkue9ktbpe5'))
+----
+POINT (20.012345 -20.012345)
+
+query T
+SELECT ST_AsText(ST_PointFromGeoHash('w000000000000000',16))
+----
+POINT (90 0)
+
+statement error pq: st_pointfromgeohash\(\): geohash decode '----': invalid character at index 0
+SELECT ST_AsText(ST_PointFromGeoHash('----'));
```

Comment:
you can omit the semicolons here `;`

Reply:
done

Reply:
i have used make buildshort command , hopefully this will make CI work this time.